WooCommerce/WooCommerceTests/Mocks/MockCollectOrderPaymentAnalyticsTracker.swift

```diff
@@ -0,0 +1,56 @@
+import Foundation
+@testable import WooCommerce
+import Yosemite
+
+final class MockCollectOrderPaymentAnalyticsTracker: CollectOrderPaymentAnalyticsTracking {
+    var connectedReaderModel: String?
+
+    func preflightResultRecieved(_ result: CardReaderPreflightResult?) {
```

Comment:
As a non-native feel free to ignore this, should it be `preflightResultReceived` ?

Reply:
Ugh I don't know why I always get this wrong! Thank you.